src/convertToObject.js

```diff
@@ -6,7 +6,21 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const updatedArray = sourceString
+    .split(';')
+    .map((item) => {
+      return item.split(':');
+    })
+    .filter((item) => item.length > 1)
+    .map((item) => {
+      return [item[0].trim(), item[1].trim()];
+    });
```

Comment:
[CODE STYLE] - All chained methods called on the same level should have the same indentation. This makes code much more readable. Align the method chains correctly to enhance readability.